sunsynk-power-flow-card/sunsynk-power-flow.js

```diff
@@ -427,14 +427,14 @@ class SunsynkPowerFlowCard extends LitElement {
+            <svg xmlns="http://www.w3.org/2000/svg" id="bat-high" x="73.5" y="279.25" width="82" height="82" preserveAspectRatio="none" opacity="${parseInt(stateObj12.state) > '80' ? '1' : '0'}" viewBox="0 0 24 24"><path fill="#f3b1c9" d="M12 20H4V6h8m.67-2H11V2H5v2H3.33C2.6 4 2 4.6 2 5.33v15.34C2 21.4 2.6 22 3.33 22h9.34c.74 0 1.33-.59 1.33-1.33V5.33C14 4.6 13.4 4 12.67 4M11 16H5v3h6v-3m0-9H5v3h6V7m0 4.5H5v3h6v-3M23 10h-3V3l-5 10h3v8"/></svg>
```

Comment:
I moved the battery lower so please revert the y values. They should all be y="290" for the 4 svg's